README.md

```diff
@@ -41,6 +41,13 @@ FingerprintJS Pro Svelte SDK is an easy way to integrate [Fingerprint Pro](https
 
```

Comment:
Consider adding a first sentence introducing fingerprint in general per: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#9c47b1318fa647d6af91254004bdb9b8 

[Fingerprint](https://fingerprint.com/) is a device intelligence platform offering 99.5% accurate visitor identification.